Synview-backend/prisma/migrations/20250625183421_init/migration.sql

```diff
@@ -0,0 +1,135 @@
+-- CreateEnum
+CREATE TYPE "Role" AS ENUM ('USER', 'MENTOR');
+
+-- CreateEnum
+CREATE TYPE "Status" AS ENUM ('PENDING', 'PROGRESS', 'DONE');
+
+-- CreateTable
+CREATE TABLE "User" (
+    "UserId" SERIAL NOT NULL,
+    "username" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
+    "passwordHash" TEXT NOT NULL,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "role" "Role" NOT NULL DEFAULT 'USER',
+
+    CONSTRAINT "User_pkey" PRIMARY KEY ("UserId")
+);
+
+-- CreateTable
+CREATE TABLE "Project" (
+    "ProjectId" SERIAL NOT NULL,
```

Comment:
Is this generated code? You should consolidate your column naming conventions to a single style. For SQL, snake case for both table and column names is most common. If it is generated from Prisma, you should consolidate there.